src/serviceconnector-passwordless/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.3.14
```

Comment:
Now we support new versioning for extension and 0.x.x is not recommended any longer. The version will start from 1.0.0 (stable) or 1.0.0b1 (preview). You can find more details in https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md.

Reply:
Updated. Thanks for reminding